Chunk mp4 + tests

dieghernan opened the pull request:
`mp4` is always chunked now. Also tests for media (and extra files for testing) added

hadley commented on tests/testthat/test-post.R:
```diff
@@ -1,12 +1,42 @@
+  path <- test_path("tweet.mp4")
+  fs <- file.size(path)
+
+  ## test-unchunked
```
Isn't this always chunked?

dieghernan replied:
Agree, I was just double-checking

hadley commented on tests/testthat/test-post.R:
```diff
@@ -1,12 +1,42 @@
-  id <- upload_media_to_twitter(test_path("tweet.gif"))
+
+  # gif
+  path <- test_path("tweet.gif")
```
I pulling `path` out into a separate variable makes the test harder to understand because it introduces indirection.

hadley commented on tests/testthat/test-post.R:
```diff
@@ -1,12 +1,42 @@
+
+  # gif
+  path <- test_path("tweet.gif")
+  fs <- file.size(path)
```
Similarly, I think using a small constant makes it easier to reason about.

hadley commented on tests/testthat/test-post.R:
```diff
@@ -1,12 +1,42 @@
+  expect_type(id, "character")
+
+  # png
+  path <- test_path("tweet.png")
```
Unless you think there's something materially different in the treatment of pngs and gifs, I think you should omit this test as it doesn't actually touch any more lines of code in this package.

dieghernan replied:
Ok, I am leaving just gif and removing also `tweet.png` file

hadley reviewed (commented):
Thanks! I've made a few comments on your test to improve readability.

dieghernan commented:
I think I addressed all your comments, now there are just 3 tests (`gif` chunked and unchunked and `mp4` test for forced chunking). I also removed the `png` as it is not used any more.

Thanks

hadley commented:
Looks great, thanks!
